cookbook-react/src/app/components/Suspense/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { Suspense } from 'react';
 
-import Loading from '../Spinner/components/loading';
+import Loading from '../../../recipes/HOCs/Spinner/components/Loading';
```

Comment:
I'd move `Loading` to another recipe (another component) if it's meant to be used separately.